plugins/modules/ec2_instance.py

```diff
@@ -55,6 +55,22 @@
         Only required when instance is not already present.
     default: t2.micro
     type: str
+  count:
+    description:
+      - Number of instances to launch.
+    default: 1
```

Comment:
If might be better not to set a default here, and *if* count is set, then we always launch a new instance.

Otherwise you can't tell the difference between count not being specified (the existing behaviour would be expected) and "always launch 1 new instance for me".

While the behaviour wouldn't be the same as ec2, this is a different module and it's already not 100% compatible.